### PR TITLE
Publish issue 14

### LINKED ISSUE #14: Change IPMX Controller to IPMX Device Controller
Capturing some discussions from the EDID Slack channel for input from Gareth in this issue  to track further discussions and changes to the Arch document. 

Gareth S-B (Sony)  2:50 PM
I know there has been some to-and-fro about the naming. I do find IPMX Controller a little confusing myself. We tend to use NMOS Client, Control System and Controller as close synonyms in specs and other material at the moment. If I understand correctly, the proposed IPMX Controller is something with an API living alongside/within a Device. Would 'IPMX Device Control' (or perhaps 'IPMX Device Controller') work as a name, more clearly distinct from IPMX Control System? (edited) 

Randy Godwin  11:08 AM

I like IPMX Device Controller. It would resolve some of the confusion associated with the user-stories that involve control of IS-05 connections. It also would fit for some of the "out-of-scope but considered" User-Stories we have in the Google Doc but not included in the architecture document.  CEC, HDCP, etc would all likely be associated with an IPMX Device Controller. Likely in an implementation though the IPMX Device Controller would also be responsible for doing logic such as parsing out a monitor's EDID and populating receiver caps according to a Phase II Best Practices,etc. All those things seem to fit with a name IPMX Device Controller.

Gareth S-B (Sony)  11:13 AM
Calling the entity an IPMX Device Controller sounds good to me - it's a doer - as distinct from an API defined for that entity, which might be the IPMX Device Control (no -er) API? (if nothing pithier comes to mind!)

### PR DESCRIPTION
**Pull request by rbgodwin**
Name change in Architecture and diagrams.  Resolved ambiguity between a more "Broadcast Controller" IPMX Controller and the more limited element "IPMX Device Controller" which handles EDID type functionality.

### CODE REVIEW

**Review by prince-chrismc — COMMENTED** [review]
Seems like you accidentally committed two extra files
- .vs/ProjectSettings.json
- .vs/slnx.sqlite *binary files are especially bad to commit*

File: docs/1.0. Architecture.md [hunk 1]
```diff
@@ -133,14 +133,14 @@ Table 2: Sequence Diagram Steps for Figure 6
...
+| 4: Update Receivers | The IPMX Device Controller updates the IS-04 Receivers with extended receiver caps. | |
 
 ##### NMOS IS-04 IS-05 Perspective
 
 Once the IPMX Encoder and Decoder Gateways have come online and processed the EDID information as shown in Figures 5 and 6 the system is ready to process IS-05 connections.
-Figure 7 shows a sequence diagram for setting up a connection between an IPMX Encoder and two IPMX Decoders. In this illustration the Pro AV Person requests via an IPMX Control System that an output stream from the IPMX Encoder is sent to two IPMX Decoders. Table 3 describes the steps with details.
+Figure 7 shows a sequence diagram for setting up a connection between an IPMX Encoder and two IPMX Decoders. In this illustration the Pro AV Person requests via an IPMX Device Control System that an output stream from the IPMX Encoder is sent to two IPMX Decoders. Table 3 describes the steps with details.
```
Comment (prince-chrismc) [inline docs/1.0. Architecture.md:143]: I think this was the _broadcast controller_

```suggestion
Figure 7 shows a sequence diagram for setting up a connection between an IPMX Encoder and two IPMX Decoders. In this illustration the Pro AV Person requests via an IPMX Control System configures an output stream from the IPMX Encoder is sent to two IPMX Decoders. Table 3 describes the steps with details.
```

**Comment by N-Nagorny** [comment]
@rbgodwin, I checked #23 and see that it already has these changes and additional ones. I used `IPMX Device Controller`, `IPMX Device Control API` and `IPMX Control System` there and cleaned the text from any other variants of these names. I think, #23 can be accepted after we finish with `IPMX Device Controller` definition and replace the obsolete terms in diagrams. Don't you mind moving to #23?

**Comment by rbgodwin** [comment]
On #23 there are no updates to the diagrams. This pull request has updates to the diagrams to match name changes. However I think there may be a consensus to use EDID Device Control and IPMX Controller so we need one more pass.

Since we are seeing confusion using the current names I would like to get these changes in this week. I can do the changes using EDID Device Control and IPMX Controller and this will be consistent with the BCP https://specs.amwa.tv/nmos-edid-connection-management/branches/publish-bcp-mappins/docs/1.0._EDID-Mapping.html#is-05-ref

**Comment by N-Nagorny** [comment]
@rbgodwin, it seems to me that we need to raise it up during the next call and then vote using a Slack poll.

**Comment by N-Nagorny** [comment]
It was covered within another PR.

**Pull request closed without merging**